org.adempiere.base/src/org/compiere/model/PO.java

```diff
@@ -2209,8 +2209,10 @@ final protected void setUpdatedBy (int AD_User_ID)
 		set_ValueNoCheck ("UpdatedBy", Integer.valueOf(AD_User_ID));
 	}	//	setAD_User_ID
 
+	private static final String TRANSLATION_CACHE_TABLE_NAME = "PO_Trl";
+	
 	/**	Cache						*/
-	private static CCache<String,String> trl_cache	= new CCache<String,String>("PO_Trl", 5);
+	private static CCache<String,String> trl_cache	= new CCache<String,String>(TRANSLATION_CACHE_TABLE_NAME, 5, CCache.DEFAULT_EXPIRE_MINUTE, true);
```

Comment:
I think we don't have any distributed cache in iDempiere core, distributed cache are a lot slower than local caches, I tested with non-distributed and it worked fine, is there any need here for distributed?

Reply:
Without distributed cache, changes make in one sever will not auto refresh the translation cache of another server. You can test that out by running 2 instance of idempiere (with hazelcast tcp/ip join enable).

Reply:
The reset call will get transmitted to other server nodes. With that, the distributed cache is probably not necessary, will re-test to confirm.